## bugfix/core

* Now inserting a tuple with the wrong "id" field into the \_priv space will
  return the correct error (gh-6295).

